lsmb-dev

```diff
@@ -74,6 +78,8 @@ show_result() {
 	== should be available at
 	======================================
 	host         : http://${IpAddr_host}:${HostPort}
+	host_dev     : http://${IpAddr_host}:${HostDevPort}
+	mailhog      : http://${IpAddr_host}:${MailHogPort}
```

Comment:
I know you have a habit of working with the op of the host - which is different from my habit of working directly with the container IP. I have no idea if either is better, but isn't our list becoming rather inconsistent with the items at the top using one approach and the rest the other?

Reply:
Fixed